tools/onenote-migration/onenote_migrate.py

```
@@ -0,0 +1,567 @@
+except ImportError:                          # pragma: no cover
+    _MSAL = None                             # type: ignore
+
+
+# ╔══════════════════════════════════════════════════════════════╗
+# ║  Config                                                      ║
+# ╚══════════════════════════════════════════════════════════════╝
+
+class Config:
+    CLIENT_ID:  str = os.getenv("ONENOTE_CLIENT_ID", "YOUR_AZURE_CLIENT_ID")
+    TENANT:     str = os.getenv("ONENOTE_TENANT",    "consumers")
+    SCOPES:     list[str] = ["Notes.Read.All", "User.Read"]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)